### PR TITLE
Populate history.js

### PR DESCRIPTION
**Pull request by Megadynamite**
Added content and formatting to the history page of the website. 
Content input from @Twosies
Format input from @hedgehog1029

### CODE REVIEW

**Review by odensc — APPROVED** [review]

File: pages/history.js [hunk 1]
```diff
@@ -1,10 +1,62 @@
...
+			bot. Sneknet V3 uses a database of previously collected answers to
+			show the user if each answer is a human or a bot. It also
+			automatically selects if the correct answer is known.
+			<br />
+			<br />
+			<strong>August 2019 - RPAN</strong>
+			<br />
+			RPAN is a streaming event with a cap on streamers and length.
+			Snakeroom creates scripts that snipe slots and allow the user to
+			broadcast from an obs stream, bypassing the intended limitation of
```
Comment (haykam821) [inline pages/history.js:28]:
```suggestion
			broadcast from an OBS stream, bypassing the intended limitation of
```

File: pages/history.js [hunk 2]
```diff
@@ -1,10 +1,62 @@
...
+			shut down by Reddit admins.
+			<br />
+			<br />
+			<strong>April 2019 - Sequence</strong>
+			<br />
+			During sequence, users upload and vote on gifs. The most upvoted
+			gifs are strung together into a video. The Snakeroom creates Sneknet
+			V2. The new extension accepts a link to a gif on the backend and
+			then selects this gif for all users running the extension, locking
+			in gif choices without chancing success on the whims of Reddit.
```
Comment (haykam821) [inline pages/history.js:41]:
```suggestion
			in GIF choices without chancing success on the whims of Reddit.
```

File: pages/history.js [hunk 3]
```diff
@@ -1,10 +1,62 @@
...
+			They also begin streaming before the event is live, though quickly
+			shut down by Reddit admins.
+			<br />
+			<br />
+			<strong>April 2019 - Sequence</strong>
+			<br />
+			During sequence, users upload and vote on gifs. The most upvoted
+			gifs are strung together into a video. The Snakeroom creates Sneknet
+			V2. The new extension accepts a link to a gif on the backend and
+			then selects this gif for all users running the extension, locking
```
Comment (haykam821) [inline pages/history.js:40]:
```suggestion
			then selects this GIF for all users running the extension, locking
```

File: pages/history.js [hunk 4]
```diff
@@ -1,10 +1,62 @@
...
+			create &quot;Place In RPAN&quot; and &quot;RPAN Plays Pokemon&quot;.
+			They also begin streaming before the event is live, though quickly
+			shut down by Reddit admins.
+			<br />
+			<br />
+			<strong>April 2019 - Sequence</strong>
+			<br />
+			During sequence, users upload and vote on gifs. The most upvoted
+			gifs are strung together into a video. The Snakeroom creates Sneknet
+			V2. The new extension accepts a link to a gif on the backend and
```
Comment (haykam821) [inline pages/history.js:39]:
```suggestion
			V2. The new extension accepts a link to a GIF on the backend and
```

File: pages/history.js [hunk 5]
```diff
@@ -1,10 +1,62 @@
...
+			streaming from a mobile device and allowing Snakeroom members to
+			create &quot;Place In RPAN&quot; and &quot;RPAN Plays Pokemon&quot;.
+			They also begin streaming before the event is live, though quickly
+			shut down by Reddit admins.
+			<br />
+			<br />
+			<strong>April 2019 - Sequence</strong>
+			<br />
+			During sequence, users upload and vote on gifs. The most upvoted
+			gifs are strung together into a video. The Snakeroom creates Sneknet
```
Comment (haykam821) [inline pages/history.js:38]:
```suggestion
			GIFs are strung together into a video. The Snakeroom creates Sneknet
```

File: pages/history.js [hunk 6]
```diff
@@ -1,10 +1,62 @@
...
+			broadcast from an obs stream, bypassing the intended limitation of
+			streaming from a mobile device and allowing Snakeroom members to
+			create &quot;Place In RPAN&quot; and &quot;RPAN Plays Pokemon&quot;.
+			They also begin streaming before the event is live, though quickly
+			shut down by Reddit admins.
+			<br />
+			<br />
+			<strong>April 2019 - Sequence</strong>
+			<br />
+			During sequence, users upload and vote on gifs. The most upvoted
```
Comment (haykam821) [inline pages/history.js:37]:
```suggestion
			During Sequence, users upload and vote on GIFs. The most upvoted
```

File: pages/history.js [hunk 7]
```diff
@@ -1,10 +1,62 @@
...
+			gifs are strung together into a video. The Snakeroom creates Sneknet
+			V2. The new extension accepts a link to a gif on the backend and
+			then selects this gif for all users running the extension, locking
+			in gif choices without chancing success on the whims of Reddit.
+			<br />
+			<br />
+			<strong>April 2018 - Circle of Trust</strong>
+			<br />
+			In Circle of Trust, users create a circle. Each circle has a
+			passcode, a user not in the circle can use the passcode to join or
```
Comment (haykam821) [inline pages/history.js:47]:
```suggestion
			passcode; a user not in the circle can use the passcode to join or
```

File: pages/history.js [hunk 8]
```diff
@@ -1,10 +1,62 @@
...
+			betray. Joining enlarges the circle, while betraying ends it. The
+			Snakeroom creates the first Sneknet, a browser extension to automate
+			the event. The extension automatically joins a circle from a command
+			sent on the backend and adds anyone running the extension before it
+			can be betrayed.
+			<br />
+			<br />
+			<strong>March 2018 - Snakeroom Genesis</strong>
+			<br />A group founds The Snakeroom Alliance after Reddit discovers
```
Comment (haykam821) [inline pages/history.js:56]:
```suggestion
			<br />
			A group founds The Snakeroom Alliance after Reddit discovers
```

**Review by haykam821 — CHANGES_REQUESTED** [review]
I'd recommend properly using a series of headers and paragraphs rather than `<br /> <br />` to achieve the grouping.

**Comment by haykam821** [comment]
On wide monitors, the text stretches out, making it harder to read:

Horizontal viewport history

On mobile, it's better due to a much smaller viewport width, but regardless, the page might need some readability improvements.

File: pages/history.js [hunk 9]
```diff
@@ -1,10 +1,62 @@
...
+			automatically selects if the correct answer is known.
+			<br />
+			<br />
+			<strong>August 2019 - RPAN</strong>
+			<br />
+			RPAN is a streaming event with a cap on streamers and length.
+			Snakeroom creates scripts that snipe slots and allow the user to
+			broadcast from an obs stream, bypassing the intended limitation of
+			streaming from a mobile device and allowing Snakeroom members to
+			create &quot;Place In RPAN&quot; and &quot;RPAN Plays Pokemon&quot;.
```
Comment (haykam821) [inline pages/history.js:30]:
```suggestion
			create &quot;Place Live&quot; and &quot;RPAN Plays Pokémon&quot;.
```

File: pages/history.js [hunk 10]
```diff
@@ -1,10 +1,62 @@
...
+
 export default () => (
 	<>
 		<Head>
 			<title>History - The Snakeroom</title>
 		</Head>
+		<Body>
+			<strong>April 2020 - Imposter</strong>
+			<br />
+			Imposter is a Turing Test, the player receives four messages from
```
Comment (haykam821) [inline pages/history.js:17]:
```suggestion
			Imposter was a Turing test where the player received four messages from
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

File: pages/history.js (hunk 8 above)
Comment (Megadynamite) [reply pages/history.js:56]: We use Prettier and ESLint to enforce code style.

File: pages/history.js (hunk 10 above)
Comment (Megadynamite) [reply pages/history.js:17]: This entire article is written in present tense as would be expected for a review. A review is conducted with the implication that the quoted source material is timeless and therefore referred to in the present tense regardless of when the review is written or read

Comment (haykam821) [reply pages/history.js:17]: [Wikipedia disagrees.](https://en.wikipedia.org/wiki/Place_(Reddit))

Comment (Megadynamite) [reply pages/history.js:17]: I am not concerned in which tense Wikipedia references Place.

Comment (Megadynamite) [reply pages/history.js]: The replacement of a comma with the word where has been applied, the change in tense has not.

**Comment by Megadynamite** [comment]
Implemented into #7 with corrected formatting by @haykam821.

**Pull request closed without merging**